pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -112,7 +117,7 @@ type NetworkHandler interface {
 	NftablesAppendRule(proto iptables.Protocol, table, chain string, rulespec ...string) error
 	NftablesLoad(proto iptables.Protocol) error
 	GetNFTIPString(proto iptables.Protocol) string
-	CreateTapDevice(tapName string, queueNumber uint32, launcherPID int, mtu int, tapOwner string) error
+	CreateTapDevice(tapName string, queueNumber uint32, launcherPID int, mtu int, owner tapOwner) error
```

Comment:
Currently the group and user id are the same. So I don't see an reason for doing this change now.

Reply:
The fact that the UID & GID are the same is a simple coincidence. 

But OK, I give up, I have used this argument already in https://github.com/kubevirt/kubevirt/pull/5417#issuecomment-818643289 and it didn't have an impact.

Reply:
The correct thing that will help the future developer to understand what are those user/group ids, is reading it from the libvirt process.
Currently we hardcode it, so having separate uid and gid or one const representing both seems equally bad to me.